meetings/2024/LDM-2024-04-17.md

```diff
@@ -2,33 +2,34 @@
 
 ## Agenda
 
-* [Relax Add requirement for collection expression conversions to types implementing IEnumerable](https://github.com/dotnet/csharplang/issues/8034) (Chuck)
-* Extensions (Julien)
-* Collection expressions (Chuck)
+*  [Relax `Add` requirement for collection expression conversions to types implementing `IEnumerable`](#relax-add-requirement-for-collection-expression-conversions-to-types-implementing-ienumerable)
+* [Extensions](#extensions)
 
-## Relax Add requirement for collection expression conversions to types implementing IEnumerable
+## Relax `Add` requirement for collection expression conversions to types implementing `IEnumerable`
+
+[#8034](https://github.com/dotnet/csharplang/issues/8034)
```

Comment:
The same link is included beneath *Discussion*. Consider removing one of the two.